python/Ganga/Runtime/bootstrap.py

```diff
@@ -1069,12 +1070,12 @@ def convert_merger_to_postprocessor(j):
 
         # JobTree
         from Ganga.Core.GangaRepository import getRegistry
-        jobtree = GPIProxyObjectFactory(getRegistry("jobs").getJobTree())
+        jobtree = getRegistry("jobs").getJobTree()
```

Comment:
getJobTree is defined within the job registry is it not?
If you're referring to the 'exportToGPI' does all of the difficulties of wrapping modules/objects/classes/my-sanity in a proxy for you now so you don't have to add proxies to the object before passing it to here as this layer expects to always add a proxy when the object in question is being exposed to the user.